### PR TITLE
ENH: Compatibility with sphinx-book-theme==1.0.0

### PR DESCRIPTION
**Pull request by AakashGfude**
Updating `sphinx-book-theme` to 1.0.0 , and other relevant packages as well

### CODE REVIEW

**Comment by AakashGfude** [comment]
@mmcky I have pushed the https://github.com/QuantEcon/lecture-python-intro/issues/57 fix here, and tested with the python advanced lecture.

**Comment by AakashGfude** [comment]
Table styling has been done in this PR. To prevent overlap of columns

**Comment by AakashGfude** [comment]
@mmcky this PR is ready to be reviewed, it has `sphinx-book-theme` to the latest version

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Fetch lecture-python-programming.myst
         shell: bash -l {0}
         run: |
-          git clone https://github.com/QuantEcon/lecture-python-programming.myst
+          git clone --branch testing-latest-jb https://github.com/QuantEcon/lecture-python-programming.myst
```
Comment (mmcky) [inline .github/workflows/ci.yml:12]: - [ ] tidy this `CI` request once merged

**Review by mmcky — APPROVED** [review]
thanks @AakashGfude this is looking good to me. Our lectures are pinned -- please go ahead and setup a new release when you are ready. I think there is just one adjustment to be made before merge.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (AakashGfude) [reply .github/workflows/ci.yml:12]: Great. Thanks @mmcky

**Comment by AakashGfude** [comment]
Merging this one. Will update `lecture-python-programming` to reflect the latest changes.

**Pull request merged**